src/store/modules/kly/kly-getters.js

```diff
@@ -5,12 +5,17 @@ export default {
   ...baseGetters,
 
   fee: (state) => (amount, recipientAddress, data, isNewAccount) => {
-    return estimateFee({
-      amount,
-      data,
-      isNewAccount,
-      nonce: state.nonce
-    })
+    try {
+      return estimateFee({
+        amount,
+        data,
+        isNewAccount,
+        nonce: state.nonce
+      })
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    } catch (error) {
+      return 0
+    }
```

Comment:
Just omit the `error` argument from catch block. ESLint comment not needed.

```suggestion
    } catch {
      return 0
    }
```